### PR TITLE
External minio

### PR DESCRIPTION
**Pull request by sthapa**
Add support for using an  external minio object store

### CODE REVIEW

**Comment by sthapa** [comment]
Should be ready for review now, I had to add/change a few lines to get this working under python 2 as well as python 3

File: servicex/transformer/object_store_manager.py [hunk 1]
```diff
@@ -30,16 +30,22 @@
 
 class ObjectStoreManager:
 
-    def __init__(self, url=None, username=None, password=None):
+    def __init__(self, url=None, username=None, password=None, use_https=False):
         from minio import Minio
-
+        if 'MINIO_SECURED' in os.environ:
+            if os.environ['MINIO_SECURED'].lower() == "true":
```
Comment (BenGalewsky) [inline servicex/transformer/object_store_manager.py]: This could just be 
```
secure_connection = os.environ['MINIO_SECURED'].lower() == "true"
```

**Review by BenGalewsky — COMMENTED** [review]
This really should be two PRs - the log formatting changes have nothing to do with external Minio

File: servicex/transformer/object_store_manager.py (hunk 1 above)
Comment (sthapa) [reply servicex/transformer/object_store_manager.py]: Wouldn't that return a KeyError is the MINIO_SECURED variable is not in the environment?

Comment (BenGalewsky) [reply servicex/transformer/object_store_manager.py]: Not if you keep the `if 'MINIO_SECURED' in os.environ:` or switch to `os.environ.get("MINIO_SECURED", "false")`

**Review by BenGalewsky — APPROVED** [review]
Ok

File: servicex/transformer/object_store_manager.py (hunk 1 above)
Comment (BenGalewsky) [reply servicex/transformer/object_store_manager.py]: Ok - it was painful that this branch caused code coverage to drop. I fixed it...

**Pull request merged**